dotnet-interactive.Tests/LspSerializationContractTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Runtime.CompilerServices;
+using Assent;
+using FluentAssertions;
+using Microsoft.DotNet.Interactive.App.Lsp;
+using Microsoft.DotNet.Interactive.Extensions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Microsoft.DotNet.Interactive.App.Tests
+{
+    public class LspSerializationContractTests
+    {
+        private readonly Configuration _configuration;
+
+        public LspSerializationContractTests()
+        {
+            _configuration = new Configuration()
+                .UsingExtension("json");
+
+            _configuration = _configuration.SetInteractive(true);
+        }
+
+        private void Descent<T>(T expected, JsonSerializer serializer, [CallerMemberName] string testName = null, [CallerFilePath] string filePath = null)
```

Comment:
I needed the exact opposite of an Assent test, and since I didn't want to mess with escaping double quotes inline in the code, I cheated and did what (I assume) Assent does, but backwards by finding an appropriate file on disk that will be deserialized and compared to a known object.  This method doesn't, however, show the diff like Assent does.

Reply:
Ok... but the opposite of Assent is Dissent.

---
In reply to: [409881202](https://github.com/dotnet/interactive/pull/348#discussion_r409881202) [](ancestors = 409881202)

Reply:
Good call, I was interpreting this as "Ascent".  Will fix.